overlord/ifacestate/helpers.go

```diff
@@ -719,3 +726,41 @@ func ensureSystemSnapIsPresent(st *state.State) error {
 	_, err := snapstate.CoreInfo(st)
 	return err
 }
+
+func hotplugTaskSetAttrs(task *state.Task, deviceKey, ifaceName string) {
+	task.Set("device-key", deviceKey)
```

Comment:
I think I missed this in previous reviews: these should all be hotplugKey and "hotplug-key" as well, so it's consistent across the board.

It's also not yet clear how these will be used, so we may still need to revisit these attributes after the real task show up to account for context, etc.